src/test/kotlin/no/nav/syfo/behandlendeenhet/api/internad/BehandlendeEnhetApiTest.kt

```diff
@@ -1,662 +1,712 @@
+                        bearerAuth(validToken)
+                    }
+                    assertEquals(HttpStatusCode.BadRequest, response.status)
+                }
+            }
+
+            @Test
+            fun `should return status BadRequest if NAV_PERSONIDENT_HEADER with invalid PersonIdent is supplied`() {
+                testApplication {
+                    val client = setupApiAndClient()
+                    val response = client.get(behandlendeEnhetUrl) {
+                        bearerAuth(validToken)
+                        header(NAV_PERSONIDENT_HEADER, ARBEIDSTAKER_PERSONIDENT.value.drop(1))
+                    }
+                    assertEquals(HttpStatusCode.BadRequest, response.status)
+                }
+            }
+
+            @Test
+            fun `should return status Forbidden if NAVIdent is denied access`() {
+                testApplication {
+                    val validTokenNoAccess = generateJWT(
+                        audience = externalMockEnvironment.environment.azureAppClientId,
+                        issuer = externalMockEnvironment.wellKnownInternalAzureAD.issuer,
+                        navIdent = VEILEDER_IDENT_NO_ACCESS,
+                    )
+                    val client = setupApiAndClient()
+                    val response = client.get(behandlendeEnhetUrl) {
+                        bearerAuth(validTokenNoAccess)
+                        header(NAV_PERSONIDENT_HEADER, ARBEIDSTAKER_PERSONIDENT.value)
+                    }
+                    assertEquals(HttpStatusCode.Forbidden, response.status)
+                }
+            }
+        }
+    }
+
+    @Test
+    fun `Get mulige oppfolgingsenheter`() {
+        testApplication {
+            repository.createOppfolgingsenhet(
+                personIdent = ARBEIDSTAKER_PERSONIDENT,
+                enhetId = EnhetId(UNDERORDNET_NR),
+                veilederident = VEILEDER_IDENT,
+            )
+            val client = setupApiAndClient()
+            val response = client.get(tilordningsenheterUrl) {
+                bearerAuth(validToken)
+            }
+            assertEquals(HttpStatusCode.OK, response.status)
+            val behandlendeEnhetList = response.body<List<EnhetDTO>>()
+
+            assertEquals(3, behandlendeEnhetList.size)
+            assertEquals(EnhetId.ENHETNR_NAV_UTLAND, behandlendeEnhetList[0].enhetId)
+            // UNDERORDNET_NR (0103) kommer foran GEOGRAFISK_ENHET_NR_2 (0102) fordi veileder har brukt 0103 en gang
+            assertEquals(UNDERORDNET_NR, behandlendeEnhetList[1].enhetId)
+            assertEquals(GEOGRAFISK_ENHET_NR_2, behandlendeEnhetList[2].enhetId)
+        }
+    }
+
+    @Nested
+    @DisplayName("Get tildelthistorikk for PersonIdent")
+    inner class GetTildelthistorikkForPersonIdent {
+        private val objectMapper = ObjectMapper()
+
+        init {
+            objectMapper.registerModule(JavaTimeModule())
+        }

```

Comment:
Det kan ikke skrives slik? 😊 `init` vil vel bare kjøres én gang på lik linje med instansieringen av private proeprties? 
```suggestion
        private val objectMapper = ObjectMapper().registerModule(JavaTimeModule())
```